caffe2/contrib/script/compiler.cc

```diff
@@ -251,6 +276,9 @@ struct DefCompiler {
     } else if (type_ident == "LL") {
       dtype->set_i(TensorProto_DataType_INT64);
       value->set_i(v);
+    } else if (type_ident == "b") {
```

Comment:
Is this only an implementation detail in the compiler or in the language can I pass, for ex, 1234b and that will emit a ConstantFill(type=bool, value=1i)?

Reply:
It is an implementation detail, True generates (TK_CONST 1 b). I will probably remove this tomorrow in favor of making it (TK_CONST "True" b), (TK_CONST "1.0" "f"), (TK_CONST "34" "I"), etc.